### PR TITLE
Adds dynamic rln relay config options

### PR DESCRIPTION
**Pull request by staheri14**
Reads the config option related to the dynamic rln-relay.
Please also see https://github.com/status-im/infra-nim-waku/pull/61

### CODE REVIEW

File: templates/docker-compose.yml.j2 [hunk 1]
```diff
@@ -56,6 +56,9 @@ services:
 {% endif %}
 {% if "rln-relay" in nim_waku_protocols_enabled %}
       --rln-relay-content-topic={{ nim_waku_rln_relay_content_topic }}
+      --rln-relay-dynamic={{ nim_waku_rln_relay_dynamic }}
+      --eth-account-address={{ nim_waku_eth_account_address }}
+      --eth-mem-contract-address={{ nim_waku_eth_mem_contract_address }}
```
Comment (jakubgs) [inline templates/docker-compose.yml.j2:61]: If those flags are related to `rln-relay` why are they not prefixed with `--rln-relay-`?

**Review by jakubgs — CHANGES_REQUESTED** [review]
The flag naming isn't great. Also, You need defaults for those variables.

File: templates/docker-compose.yml.j2 (hunk 1 above)
Comment (rymnc) [reply templates/docker-compose.yml.j2:61]: Tracking that issue here - https://github.com/status-im/nwaku/issues/1196

**Comment by staheri14** [comment]
> The flag naming isn't great. Also, You need defaults for those variables.

@jakubgs 
Thanks, are the default values different than the ones already defined in this PR https://github.com/status-im/infra-nim-waku/pull/61? Let me know if they need to be defined somewhere else? Thanks!

**Review by rymnc — APPROVED** [review]
LGTM!

**Review by jakubgs — CHANGES_REQUESTED** [review]
@staheri14 yes, I'd say the defaults from the binary should be used as defaults in the Ansible role as well.

It's important to keep in mind that the `defaults/main.yml` also acts as documentation for what options are available. Otherwise user would have to scour the whole role to find what variables exist and can be used.

**Comment by staheri14** [comment]
> It's important to keep in mind that the defaults/main.yml also acts as documentation for what options are available. Otherwise user would have to scour the whole role to find what variables exist and can be used.

Thanks, added the available options to the `defaults/main.yml`  https://github.com/status-im/infra-role-nim-waku/pull/5/commits/9fb80e587560a86ee95e51a2da025a2a0752ad6b

**Review by jakubgs — APPROVED** [review]
LGTM. Please squash and use the commit in the other PR.

**Pull request merged**